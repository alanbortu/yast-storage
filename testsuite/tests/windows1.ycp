// testedfiles: helper1b.yh
{
    include "setup-system.yh";

    setup_system("windows");

    include "helper1a.yh";

    import "ProductFeatures";

    ProductFeatures::SetBooleanFeature("partitioning", "try_separate_home", true);
    ProductFeatures::SetBooleanFeature("partitioning", "proposal_lvm", false);
    ProductFeatures::SetStringFeature("partitioning", "root_max_size", "20 GB");
    ProductFeatures::SetStringFeature("partitioning", "root_base_size", "15 GB");

    include "helper1b.yh";
}
