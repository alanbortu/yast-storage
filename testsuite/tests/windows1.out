Dump	Proposal:
Dump	Create extended partition /dev/sda2 (34.52 GiB)
Dump	Create swap volume /dev/sda5 (517.72 MiB)
Dump	Create root volume /dev/sda6 (15.01 GiB) with ext4
Dump	Create volume /dev/sda7 (19.01 GiB) for /home with ext4
